src/deploy/Network.ts

```
@@ -245,6 +237,31 @@ export async function deployNetworkComet(
     configuratorProxy = await deploymentManager.contract('configurator') as Configurator;
   }
 
+
+  if (shouldDeploy(contractsToDeploy.all, contractsToDeploy.rewards)) {
+    rewards = await deploymentManager.deploy<CometRewards, CometRewards__factory, [string]>(
+      'CometRewards.sol',
+      [adminSigner.address]
+    );
+
+    // Set the rewards config for Comet in CometRewards
+    // XXX we should validate rewardTokenAddress is set earlier
```

Planning on adding some input validation so we can fail early.

---

Done